vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -105,3 +105,238 @@ To add a `Figure` to your page:
 
 If the existing figure functions from [`vizro.figures`](../API-reference/figure-callables.md) do not serve your purpose,
 there is always the option to create a [custom Figure](custom-figures.md).
+
+
+### KPI Cards
+A KPI card is a dynamic card that can display a single value. Optionally, a title, icon and reference value can be added.
+It is a common visual component to display key metrics in a dashboard. Vizro supports two pre-defined KPI card functions:
+
+- [`kpi_card`](../API-reference/figure-callables.md#kpi_card): A KPI card with a single value.
+- [`kpi_card_with_reference`](../API-reference/figure-callables.md#kpi_card_with_reference): A KPI card with a single value and a reference value.
+
+#### Existing KPI Card functions
+!!! example "KPI Card Variations"
+
+    === "app.py"
+        ```py
+        import pandas as pd
+        import vizro.models as vm
+        from vizro import Vizro
+        from vizro.figures import kpi_card, kpi_card_reference
+
+        df = pd.DataFrame([[67434, 65553, "A"], [6434, 6553, "B"], [34, 53, "C"]], columns=["Actual", "Reference", "Category"])
```

Comment:
These values seem kind of random/complicated - I would simplify them to just like 1, 2, 3 or the like. See other comment on a suggested dataframe that will achieve clearer positive vs. negative demonstration also.